### DISCUSSION TITLE
Uncertainty in views in the blacklitterman_stats

### DISCUSSION THREAD

**Discussion by aliencaocao** [dcajasn/Riskfolio-Lib#116] [Q&A] [answered]
Hi, I want to use the black litterman model for portfolio, but according to the equation, there should be this omega term that is a matrix of the uncertainty of each view. However, I have no idea how to specify this uncertainty in riskfolio lib, using the assets_views function or any other function. The example code only have 'return'. Thank you

![image](https://user-images.githubusercontent.com/20109683/197384105-717e9221-0588-4582-990b-096c45645663.png)
Talking about the term "the covariance matrix of the error views." here

**Comment by dcajasn** [comment, marked as answer]
Hi @aliencaocao, 

That parameter is calculated following the paper specified in docs.

Best,
Dany

**Reply by aliencaocao** [reply]
Hi, the first paper (B4) is paid, so can I just double confirm its calculated in this way:
![image](https://user-images.githubusercontent.com/20109683/197396572-45c9d594-c4ae-437c-95c4-d45c179109b7.png)
S is covariance matrix of asset returns from historical data
Thanks!

**Reply by dcajasn** [reply]
For paid papers try sci-hub.
